index.bs

```diff
@@ -422,14 +425,22 @@ it shouldn't be used in practice
 as it is easy to filter out the added noise.
 
 
-<h4 id="informing-user">Keeping the user informed about API use</h4>
+<h4 id="inform-user">Keep the user informed about API use</h4>
 
 User agents may choose to keep the user informed
 about current and past use of the API.
 
 Note: this does not imply keeping a log of the actual [=sensor readings=]
 which would have issues of its own.
 
+<h4 id="input-elements">Input Elements</h4>
+
+When an <{input}> element, whose [=browsing context container=]'s [=node document=]'s
+[=origin=] is not [=same origin-domain=] with its [=parent browsing context=]'s document's
+[=origin=], is [=focused=], the user agent should [=stop the sensor=], or alternatively,
+[=reduce accuracy=] and [=limit maximum polling frequency=], to mitigate against PIN
+skimming attacks that try to correlate the user's input with sensor readings.
```

Comment:
Please see the document in #192 that lists known threats. Specifically, PIN skimming is discussed in more detail in:

>[11] Spreitzer, Raphael. "Pin skimming: Exploiting the ambient-light sensor in mobile devices." Proceedings of the 4th ACM Workshop on Security and Privacy in Smartphones & Mobile Devices. ACM, 2014. URL: https://arxiv.org/pdf/1405.3760.pdf